### DISCUSSION TITLE
OS 3 easy pieces course lectures

### DISCUSSION THREAD

**Discussion by dvirberlo** [spamegg1/reviews#7] [Ideas]
Hi,
I started to read through your reviews.
Quite nice, overall.

You wrote how much you enjoyed the book "Operating Systems - three easy pieces".
Just wanted to mention that in the course's website there are links to AMAZING video lectures on YouTube, by the author of the book.
(I think the book is written by the professor and his wife)

For future readers, maybe consider mentioning the lectures...

Good job 😉

**Comment by spamegg1** [comment]
Thanks! I believe you are talking about this?
https://pages.cs.wisc.edu/~remzi/Classes/537/Spring2018/Discussion/videos.html

I am aware of the lectures. They are included in the [OSSU page for the course](https://github.com/ossu/computer-science/blob/master/coursepages/ostep/README.md). 

I hadn't included them here, because I hadn't used those videos myself (so it would be dishonest to write a review of them). Also the videos contain some stuff not in the book (like the projects, or talking about classroom/homework/grading related issues).

After looking at them a bit, personally I did not like the videos, but I'll think about including a link for other readers who might find it potentially useful.

Thanks again!

**Comment by spamegg1** [comment]
Handled here 146c89a702b992065a36b6ec95a3f191f1a2bab0
